### DISCUSSION TITLE
useState, useTextController or any use methods not showing in hooks_riverpod

### DISCUSSION THREAD

**Discussion by rirjkl19** [rrousselGit/riverpod#1995] [Q&A] [answered]
Hi I'm new to riverpod but I had experience using provider package with hooks package. Its convenient for me to use hooks for use cases such as `useState`, `useTextController` however. Moving to riverpod using hooks_riverpod.  It seems like the `useState` and `useTextController` isn't being suggested in the linter. I've been scrounging hooks_riverpod package on where this global methods are declared but I can't seem to find it.

**Comment by TimWhiting** [comment, marked as answer]
They are in the flutter_hooks package.

**Reply by rirjkl19** [reply]
Thank you!
